diva-irma.js

```diff
@@ -314,6 +315,43 @@ function getIrmaStatus(irmaSessionType, irmaSessionId) {
     });
 }
 
+function jsonQuoteNumberValues(payload) {
+  // Regex for quoting unquoted JSON number values
+  return payload.replace(/: ?(-?\d+)/g, ':"$1"');
+}
+
+function signatureFromToken(token) {
+  // Because of the large numeric values overflowing, custom parsing is needed.
```

Comment:
Can we elaborate more on what happens exactly here? I think I understand because I have some background info on the crypto, but since the data transformations are quite complex it would be good to add a clear explanation here and/or link to sources describing why the 'fix' is needed.

Reply:
How about the updated comments?

Reply:
👍